src/sentry/utils/security/orgauthtoken_jwt.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+from uuid import uuid4
+
+from django.conf import settings
+
+from sentry.utils import jwt
+
+
+def generate_token(org_slug: str):
+    jwt_payload = {
+        "iss": "sentry.io",
```

Comment:
Should this be use even when the instance is self-hosted/single-tenant?

Reply:
yeah, valid question... took this from here: https://github.com/getsentry/rfcs/pull/91/files#diff-3109d1f30b4b81085e85d841deafefb0d1b43cc5345c64514475512a4ce9fdeeR94

IMHO it should be something consistent that says "this is from sentry", and not differ when it is self-hosted/single-tenant. Not sure if `sentry.io` is the best value for this then, but I'd say it's fine, maybe (just "Sentry" may also be ambiguous, ...)